server/src/main/java/io/druid/segment/loading/SegmentLoaderLocalCacheManager.java

```diff
@@ -270,4 +264,19 @@ public void cleanupCacheFiles(File baseFile, File cacheFile) throws IOException
       }
     }
   }
+
+  public Set<StorageLocation> getSortedSet(List<StorageLocation> locs)
```

Comment:
Can this return an `Iterable` or `Collection`? An the method itself make a new `ArrayList` with the contents of `locs`, then just use `Collections.sort` and return the created array?